meta-resin-common/recipes-support/resin-expand/resin-data-expander.bb

```diff
@@ -1,35 +0,0 @@
-DESCRIPTION = "Resin data partition expander"
```

Comment:
Maybe rephrase commit log:

"we mount machine-id in initramfs which is on the resin-state logical partition in the extended primary partition)."

instead of just

"we mount machine-id in initramfs
which is on state logical partition in the extended primary partition)."